bip-ephemeralanchors.mediawiki

```diff
@@ -0,0 +1,160 @@
+<pre>
+  BIP: ?
+  Layer: Mempool Policy
```

Comment:
BIP 125 (RBF) uses "Layer: Application" here. Having a standard here is pretty much about coordination between node behaviour ("this sort of tx will be relayed") and applications ("how do i make txs that will be relayed?") so that seems reasonable?

Reply:
did @ajtowns suggestion for now